### ISSUE TITLE
[AWS Orgs] CloudFormation for EC2 instance

### ISSUE THREAD

**Issue by orestisfl** [elastic/cloudbeat#1131]
**Motivation**

The current aws-orgs CloudFormation template successfully deploys IAM roles to all organization accounts but it doesn't start an agent to actually run cloudbeat.

**Definition of done**
What needs to be completed at the end of this task
- [ ] Decide if we are going to merge the two templates together or keep them separate
- [ ] Extend the CloudFormation template to start an EC2 instance in the management account like we do for CSPM

**Out of scope**
Everything that falls into the related tickets

**Related tasks/epics**
- Follows up https://github.com/elastic/cloudbeat/issues/1036
- Blocked by https://github.com/elastic/cloudbeat/issues/1122

**Comment by CohenIdo** [comment]
8.10.0 BC1 QA cycle - verified : 


![Image](https://github.com/elastic/cloudbeat/assets/90558359/1edb9433-ed69-4d03-bf35-ff6376df1f13)
